### PR TITLE
Fix events list render bug

### PR DESCRIPTION
**Pull request by PeterJCLaw**
Per discussion on #34 (though unrelated to that issue itself)

### CODE REVIEW

**Comment by jimmythompson** [comment]
This appears to have shifted the navigation items down in the header:

screen shot 2016-10-08 at 20 24 54

**Comment by PeterJCLaw** [comment]
That should be resolved in e7a6555.

@jimmythompson do you have any guidelines on the sorts of things which should be in each of the .scss files? Perhaps you could add some comments at the top of each and/or some central documentation to clarify that?

**Comment by PeterJCLaw** [comment]
@jimmythompson This is becoming much more of an issue now that we're getting more events pages. Can you please take another look? I'm pretty sure the issue you raised with the SCSS affecting the header is fixed by https://github.com/srobo/website/pull/42/commits/e7a65552470f48c4df1eb6d866a47b8a1244d320.

File: _sass/base.scss [hunk 1]
```diff
@@ -39,6 +39,14 @@ a {
   @include media-query("medium") {
     padding: $spacing 0 (3 * $spacing);
   }
+
+  ul {
```
Comment (jimmythompson) [inline _sass/base.scss]: Moving this has messed some things around when I run this branch locally. Particularly on the home page.

**Comment by PeterJCLaw** [comment]
Hi @jimmythompson good spot, thanks. I've given up on CSS change for now, so this branch just contains the major spacing fix.

Do you have some guidelines on which CSS classes should be used where? For example, the events pages are using `article-preview` for their main content blocks, which doesn't really make sense. Is there a general class we can apply to containers of wrapped prose text? Ideally that would be the parent of the block of SCSS I've been moving around here I think.

**Comment by PeterJCLaw** [comment]
Meta: if you could ensure that in future you clarify exactly _what_ has changed (either in description or screenshots) it will make it much easier to correct things!

**Comment by PeterJCLaw** [comment]
@jimmythompson ping on this.

**Comment by jimmythompson** [comment]
Before I merge, can you just check you still get the problem in master? I played around with article styling, which included lists on event pages.

**Comment by PeterJCLaw** [comment]
That does now seem to be fixed in the styling, thanks. (I still think that we should avoid having empty lines between list entries though).

**Comment by PeterJCLaw** [comment]
No longer relevant.

**Pull request closed without merging**